src/components/ImageGuard/ImageGuard.tsx

```diff
@@ -463,13 +463,17 @@ ImageGuard.Report = function ReportImage({
   return (
     <Group
       spacing={4}
-      sx={{
-        position: 'absolute',
-        top: 5,
-        left: position === 'top-left' ? 5 : undefined,
-        right: position === 'top-right' ? 5 : undefined,
-        zIndex: 8,
-      }}
+      sx={
```

Comment:
Same here, style should be enough considering what we want doesn't take advantage of `sx`